docs/docusaurus/src/theme/DocSidebarItems/index.js

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
 import DocSidebarItems from '@theme-original/DocSidebarItems';
 import {useNavbarMobileSidebar} from "@docusaurus/theme-common/internal";
+import BrowserOnly from "@docusaurus/BrowserOnly";
 
 export default function DocSidebarItemsWrapper(props) {
     const mobileSidebar = useNavbarMobileSidebar();
     const handleMobileDocSidebarItemClick = (item) => {
-        if (!props.onItemClick) return
+        if(window.innerWidth > 996) return;
```

Comment:
let's put this value in a config var so we can make it obvious that we're referring to a mobile breakpoint
```suggestion
        if(window.innerWidth > MOBILE_BREAKPOINT) return;
```